src/core/defaults.ts

```diff
@@ -65,11 +65,6 @@ const DEFAULTS: Partial<Configuration> = {
         suffix: '',
         useReferenceKeys: true,
         navigations: {}
-      },
```

Comment:
We should still be providing the defaults here as we want them to be configurable